### ISSUE TITLE
“Where can I get the source file for BLDC_model_sf.mexw64?”

### ISSUE THREAD

**Issue by gbgms** [EFeru/hoverboard-firmware-hack-FOC#497]
### Variant

USART

### Control type

FOC

### Control mode

Voltage

### Description

 I want to get the source file for BLDC_model_sf.mexw64. If it is convenient to provide, I would be very grateful.

**Comment by Candas1** [comment]
Hi, have you already looked at this repo ?
https://github.com/EFeru/bldc-motor-control-FOC

**Comment by gbgms** [comment]
> 你好，你已经看过这个 repo 了吗？ https://github.com/EFeru/bldc-motor-control-FOC

Yes, I want to view the source code of BLDC_model_sf, but it’s only available in mexw64 format, which cannot be viewed
